runtime/Cpp/runtime/src/tree/xpath/XPathLexer.cpp

```diff
@@ -155,7 +155,7 @@ void XPathLexer::action(RuleContext *context, size_t ruleIndex, size_t actionInd
   }
 }
 
-void XPathLexer::IDAction(antlr4::RuleContext *context, size_t actionIndex) {
+void XPathLexer::IDAction(antlr4::RuleContext */*context*/, size_t actionIndex) {
```

Comment:
This file is generated. I added a comment #4171 (which is about a similar issue) to fix the generator template instead.

Reply:
Great, many thanks!